ipn/ipnlocal/cert.go

```diff
@@ -124,37 +139,40 @@ func (b *LocalBackend) GetCertPEM(ctx context.Context, domain string) (*TLSCertK
 func (b *LocalBackend) shouldStartDomainRenewal(cs certStore, domain string, now time.Time, pair *TLSCertKeyPair) (bool, error) {
 	renewMu.Lock()
 	defer renewMu.Unlock()
-	if last, ok := lastRenewCheck[domain]; ok && now.Sub(last) < time.Minute {
-		// We checked very recently. Don't bother reparsing &
-		// validating the x509 cert.
-		return false, nil
+	if last, ok := lastRenewCheck[domain]; ok && now.Before(last.cacheUntil) {
+		// Don't bother reparsing & validating the x509 cert until we know it's
+		// needed.
+		return last.res, nil
```

Comment:
yeah you're only using it here, and this an be replaced by `now.Before(renewAt)`